plugin/src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -604,7 +604,7 @@ private void recalculateIsland(@NotNull Island island, @NotNull List<Chunk> chun
                             for (int x = 0; x < 16; x++) {
                                 for (int z = 0; z < 16; z++) {
                                     if (island.isInIsland(x + (chunk.getX() * 16), z + (chunk.getZ() * 16))) {
-                                        final int maxy = chunk.getHighestBlockYAt(x, z);
+                                        final int maxy = Math.min(255, chunk.getHighestBlockYAt(x, z));
                                         for (int y = 0; y <= maxy; y++) {
```

Comment:
Maybe remove the `=` here instead of using the Math.min() method

Reply:
This wont work cuz spigot is dum and slightly changed this method at one point, it used to be the highest block, now its the block above the highest block

Reply:
And removing the = would only count until one block below so it would work again 🤔